### ISSUE TITLE
Action times out

### ISSUE THREAD

**Issue by tillkruss** [cloudflare/wrangler-action#71]
Similar to #44 we're currently unable to deploy our site because the workflow never finishes:

```
Using namespace for Workers Site "...-workers_sites_assets"
 Success
 Uploading site files
 Successfully published your script to
 .../* => stayed the same
with this schedule
 */6 * * * *
 */15 * * * *
 */20 * * * *
 Deleting stale files...
Error: The operation was canceled.
```

It runs for 10 minutes until it times out, because of our:

```yaml
timeout-minutes: 10
```

The wrangler version is v1.19.10.

**Comment by tillkruss** [comment]
Setting `wranglerVersion: '1.19.8'` works just fine.

**Comment by threepointone** [comment]
This should be fixed in 1.19.11
